### DISCUSSION TITLE
Pimcore 11: Pimcore continue showing loading icon when drag and drop object item in folder

### DISCUSSION THREAD

**Discussion by sandyHM** [pimcore/pimcore#17159] [Q&A / Support] [answered]
Pimcore continue showing loading icon when drag and drop object item in folder that contain more then 100 object items. if I expand that folder in which I drag and drop the object , then process is successful but if I not expand the folder in which i want to drag and drop the object item pimcore showing loading.

Step to reproduce:
1. Open object .
2. drag and drop object item to another folder that not expand yet.
3. continue showing loading
![pimcore loading](https://github.com/pimcore/pimcore/assets/107986459/09eb0e6b-4bad-4187-a4bd-2148676cc70c)

**Comment by kingjia90** [comment]
Thank you for reporting, looks like some bug, opened an issue on https://github.com/pimcore/admin-ui-classic-bundle/issues/563

**Reply by kingjia90** [reply, marked as answer]
Resolved by https://github.com/pimcore/admin-ui-classic-bundle/pull/757

**Comment by sandyHM** [comment]
Hi @kingjia90 
in mentioned scenario **onTreeNodeMove** function not working properly in /**vendor/pimcore/admin-ui-classic-bundle/public/js/pimcore/object/tree.js**
